src/App.jsx

```diff
@@ -14,70 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {!selectedGood ? (
+          'No goods selected'
+        )
+          : (
+            <>
+              {`${selectedGood} is selected`}
+              <button
+                onClick={() => setSelectedGood('')}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </>
+          )
+        }
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            good !== selectedGood ? (
+              <tr key={good} data-cy="Good">
+                <td>
+                  <button
+                    onClick={() => {
+                      setSelectedGood(good);
+                    }}
+                    data-cy="AddButton"
+                    type="button"
+                    className="button"
+                  >
+                    +
+                  </button>
+                </td>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+                <td data-cy="GoodTitle" className="is-vcentered">
+                  {good}
+                </td>
+              </tr>
+            )
+              : (
+                <tr data-cy="Good" className="has-background-success-light">
+                  <td>
+                    <button
+                      onClick={() => {
+                        setSelectedGood('');
+                      }}
+                      data-cy="RemoveButton"
+                      type="button"
+                      className="button is-info"
+                    >
+                      -
+                    </button>
+                  </td>
```

Comment:
I'd prefer using single button, with different attributes, text and onClick logic

Reply:
I've corrected the code but left two buttons. I think it will be more convenient to read this way. This issue was also raised during the morning meeting. We decided that it won't be a mistake.